Client/Frontend/Login/LoginListViewController.swift

```diff
@@ -0,0 +1,352 @@
+// Copyright 2021 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import UIKit
+import Shared
+import BraveShared
+import Storage
+import Data
+
+private let log = Logger.browserLogger
+
+class LoginListViewController: LoginAuthViewController {
+    
+    // MARK: UX
+    
+    struct UX {
+        static let headerHeight: CGFloat = 44
+    }
+    
+    // MARK: Constants
+    
+    struct Constants {
+        static let saveLoginsRowIdentifier = "saveLoginsRowIdentifier"
+    }
+    
+    // MARK: Section
+    
+    enum Section: Int, CaseIterable {
+        case options
+        case savedLogins
+    }
+    
+    weak var settingsDelegate: SettingsDelegate?
+    
+    // MARK: Private
+    
+    private let profile: Profile
+    private var loginEntries = [Login]()
+    private var isFetchingLoginEntries: Bool = false
```

Comment:
`Bool` can be removed

Reply:
Fix at https://github.com/brave/brave-ios/pull/4124/commits/ded5065d24f539a7dee6bb8bcb40b93a2b70fe7b